src/interfaces/widget.ts

```diff
@@ -31,6 +31,10 @@ export const TableLikeWidgetTypes: Array<WidgetTypes | string> = [
     WidgetTypes.PickListPopup
 ]
 
+/**
+ * @param maxInput Maximum number of characters
+ * @param hidden Whether the field is hidden
```

Comment:
TSDoc for interface properties should be placed separately for each corresponding property, i.e. `@param` is not valid for interfaces and will not be parsed correctly.

See reference:
https://github.com/microsoft/tsdoc/blob/746a8fb8fe89790107325c79e9077cdc8c83546a/playground/src/CodeEditor.tsx#L5

Reply:
I did as you are https://github.com/tesler-platform/tesler-ui/commit/2006015d86bdc0b76af3f9a1f492c877eb30a19e#diff-212d66cacab7e2fade02457e67695c21R132
I'll fix it